README.md

```diff
@@ -73,7 +73,9 @@ rm -rf $HOME/.rustup/toolchains/stable-aarch64-apple-darwin
 mv $HOME/.rustup/toolchains/1.72.1-aarch64-apple-darwin $HOME/.rustup/toolchains/stable-aarch64-apple-darwin
 ```
 
-2. If you run into OOM errors, especially while running evaluations that use LLMs as evaluators like toxicity and
+2. Due to `fmeval`'s use of [Ray](https://www.ray.io/), and the fact that [Ray on Windows is still in beta](https://docs.ray.io/en/latest/ray-overview/installation.html#windows-support), users running evaluations on Windows machines may encounter Windows-specific issues. For example, `ray.init()` may raise an OS error, as described in [this thread](https://discuss.ray.io/t/ray-init-raises-oserror-assignprocesstojobobject-failed/7018).
```

Comment:
Suggestion: Could we simplify this sentence and explicitly direct users to the Ray website to view current limitations? 

For example, "There are known limitations and issues when running `fmeval` on Windows due to `fmeval`'s use of Ray and Ray's Windows support in beta. You can view more details on limitations of running Ray on Windows on [Ray's webpage](https://docs.ray.io/en/latest/ray-overview/installation.html#windows-support)."